test/services/clients/dissolutionApi.client.test.ts

```diff
@@ -47,7 +51,53 @@ describe('DissolutionApiClient', () => {
       assert.equal(config.headers['Content-Type'], 'application/json')
       assert.equal(config.headers.Accept, 'application/json')
 
-      assert.equal(response, RESPONSE.data)
+      assert.equal(response, CREATE_RESPONSE.data)
+    })
+  })
+
+  describe('getDissolution', () => {
+    it('should return dissolution if dissolution is present in the database', async () => {
+      getStub = sinon.stub().resolves(GET_RESPONSE)
+      axiosInstance.get = getStub
+
+      const response = await client.getDissolution(TOKEN, COMPANY_NUMBER)
+
+      const reqUrl: string = `${dissolutionApiUrl}/dissolution-request/${COMPANY_NUMBER}`
+
+      assert.isTrue(getStub.called)
+
+      const [url, config] = getStub.args[0]
+      assert.equal(url, reqUrl)
+      assert.equal(config.headers.Authorization, 'Bearer ' + TOKEN)
+      assert.equal(config.headers['Content-Type'], 'application/json')
+      assert.equal(config.headers.Accept, 'application/json')
+
+      assert.equal(response, GET_RESPONSE.data)
+    })
+
+    it('should return null if dissolution is not present in the database', async () => {
```

Comment:
frontend should never refer to the database. Instead:
```
should return null if dissolution does not exist for the provided company
```

You are missing a test for what happens if a non-404 error is returned.